src/Core/src/Platform/Windows/PickerExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿#nullable enable
+using Microsoft.Maui.Graphics;
+using Microsoft.UI.Xaml.Controls;
+using WBrush = Microsoft.UI.Xaml.Media.Brush;
+
+namespace Microsoft.Maui
+{
+	public static class PickerExtensions
+	{
+		public static void UpdateTitle(this MauiComboBox nativeComboBox, IPicker picker)
+		{
+			nativeComboBox.Header = null;
+
+			nativeComboBox.HeaderTemplate = string.IsNullOrEmpty(picker.Title) ? null : 
+				(UI.Xaml.DataTemplate)UI.Xaml.Application.Current.Resources["ComboBoxHeader"];
+
+			nativeComboBox.DataContext = picker;
+		}
+		public static void UpdateTextColor(this MauiComboBox nativeComboBox, IPicker picker)
+		{
+			nativeComboBox.UpdateTextColor(picker, null);
+		}
+		
+		public static void UpdateTextColor(this MauiComboBox nativeComboBox, IPicker picker, WBrush? defaultForeground)
+		{
+			Color color = picker.TextColor;
+			nativeComboBox.Foreground = color.IsDefault() ? (defaultForeground ?? color.ToNative()) : color.ToNative();
+		}
+
+		public static void UpdateSelectedIndex(this MauiComboBox nativeComboBox, IPicker picker)
+		{
+			nativeComboBox.SelectedIndex = picker.SelectedIndex;
+		}
+
+		public static void UpdateCharacterSpacing(this MauiComboBox nativeComboBox, IPicker picker)
+		{
+			nativeComboBox.CharacterSpacing = picker.CharacterSpacing.ToEm();
+		}
+
+		public static void UpdateFont(this ComboBox nativeComboBox, IPicker picker, IFontManager fontManager) =>
```

Comment:
should this also be MauiComboBox?

```suggestion
		public static void UpdateFont(this MauiComboBox nativeComboBox, IPicker picker, IFontManager fontManager) =>
```

Reply:
this is fine because allows other people to use a extension without creating using our MauiComboBox